doc/examples/numpy_operations/plot_structuring_elements.py

```diff
@@ -0,0 +1,60 @@
+"""
+=============================
+Generate structuring elements
+=============================
+
+This example shows how to use functions in :py:module:`skimage.morphology`
```

Comment:
Try this to fix the CI failure
```suggestion
This example shows how to use functions in :py:mod:`skimage.morphology`
```